pkgs/tools/filesystems/curlftpfs/default.nix

```diff
@@ -14,6 +14,8 @@ stdenv.mkDerivation rec {
     # it is known to cause problems. Search online for "rpl_malloc" and
     # "rpl_realloc" to find out more.
     ./fix-rpl_malloc.patch
+    ./suse-bug-580609.patch
```

Comment:
Are they available somewhere online, so that we can use `fetchpatch`?

Reply:
I've unpacked them from here: http://download.opensuse.org/update/13.2/src/curlftpfs-0.9.2-61.3.1.src.rpm

They use different patch format though (`.orig` instead of `a/` and `b/`) and the patches are very small, so I'm not sure if it's justified.